pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -826,6 +826,22 @@ func trimBackingFileName(fileName string) string {
 	return backingFilePattern.ReplaceAllString(fileName, ".vmdk")
 }
 
+func (r *Builder) baseVolume(fileName string) string {
+	if r.Plan.Spec.Warm {
+		// for warm migrations, we return the very first volume of the disk
+		// as the base volume and CBT will be used to transfer later changes
+		return backingFilePattern.ReplaceAllString(fileName, ".vmdk")
```

Comment:
can we remove `trimBackingFileName` then?

Reply:
no, it's used elsewhere

Reply:
I changed this part though to call trimBackingFileName rather than duplicating its content